feat(content): Earth Day culture conversation (#9310)
endless-sky/endless-sky: data/human/culture conversations.txt

			label end
			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
				decline

# Note about Earth Day and Earth Hour:
# Earth Day (in 2023) is a day of awareness about climate change and occurs every 22nd of April. Earth Hour is an hour
# of blackout around cities worldwide, also to bring awareness to climate change, and it's observed on the last Saturday
# of March.

# Since Earth Day has become a quasi-'national day' for Earthlings in Endless Sky, it would be interesting to have Earth
# Hour be conflated with Earth Day as well, to demonstrate how people in a thousand years' time observe what we observe
# today and how its meaning has changed.
mission "Earth Day Blackout"
	minor
	landing
	source Earth
	to offer
		day = 22
		month = 5
	on offer
		conversation

> 			`As you follow the tracking beacons towards Earth's central spaceport, you realize there's something very off; what was the bustling spaceport city is now completely still, and even though it is near midnight, the supertall buildings and skyscrapers are curiously dark; not a light is on for kilometers.`

Author: ```suggestion
			`As you follow the tracking beacons towards Earth's central spaceport, you realize there's something very off. What was the bustling spaceport city is now completely still, and even though it is near midnight, all its characteristic supertall buildings and skyscrapers are curiously dark; not a light is on for kilometers.`
```
I don't understand removing the second semicolon, the first looks fair though. But the part about the supertall buildings stand alone now, so I'll also add this bit.

> 			`As you follow the tracking beacons towards Earth's central spaceport, you realize there's something very off; what was the bustling spaceport city is now completely still, and even though it is near midnight, the supertall buildings and skyscrapers are curiously dark; not a light is on for kilometers.`

Reviewer: Megascrapers are already an iconic element of overpopulated, urban, sci-fi settings. There's no need to lengthen an already long sentence to specify that it's characteristic of Earth.
```suggestion
			`As you follow the tracking beacons towards Earth's central spaceport, you realize there's something very off. What was the bustling spaceport city is now completely still, and even though it is near midnight, the supertall buildings and skyscrapers are curiously dark; not a light is on for kilometers.`
```

> 			`	However, the spaceport seems to still be operating, because the standard, automated procedures of signing landing documents, refuelling and recalibrating, and checking your ship's spaceworthiness were met and completed as you landed. But when you enter the spaceport terminal, there are wax candles spaced out to bring some light into the now dark corridors. It's a strange sight for the most populated planet in human space.`

Author: The thought was that the player would see the operational spaceport and think, 'The spaceport must still have their lights on if they can continue to service ships,' and then when they enter the terminal the lights aren't on anyway. But I think it will be better to change it.

			choice
				`	(Find someone and ask.)`
				`	(The blackout will probably pass.)`
					goto pass
			branch attendant tourist
				random > 50

			label attendant
			`	You approach an information desk in the terminal's atrium that, given the situation, doesn't seem to be as busy as expected. After a moment you get to the attendant behind the counter.`
			choice
				`	"What's with the blackout?"`
				`	"When is the blackout going to be fixed?"`

> 			`	The attendant chuckles to himself. "I should have expected that question from your accent. Today's Earth Day, when all the cities of Earth - and I mean all of them - participate in a total blackout. Even the Parliament complex was out today. It makes us think about our origins; that we once lived on the soil without electricity, all confined to just this one planet here, divided and uncertain. Of course, people like me still have to keep working, but for other people it's a day off to enjoy what little wildlife and earth us Earthlings have left, ironically. Do you have any other questions?`

Reviewer 2: ```suggestion
			`	The attendant chuckles to himself. "I should have expected that question from your accent. Today's Earth Day, when all the cities of Earth - and I mean all of them - participate in a total blackout. Even the Parliament complex was out today. It makes us think about our origins; that we once lived on the soil without electricity, all confined to just this one planet here, divided and uncertain. Of course, people like me still have to keep working, but for other people it's a day off to enjoy what little wildlife and Earth us Earthlings have left, ironically. Do you have any other questions?`
```

Reviewer 3: This could go either way, but I left it as is when I looked it over because I figured it was trying to be funny with "eart"h as in "dirt" as a nature reference than a literal concern of space on the planet Earth.

But if that joking usage isn't coming through reliably for people, probably is best to change it as suggested.

> 			`	The attendant chuckles to himself. "I should have expected that question from your accent. Today's Earth Day, when all the cities of Earth - and I mean all of them - participate in a total blackout. Even the Parliament complex was out today. It makes us think about our origins; that we once lived on the soil without electricity, all confined to just this one planet here, divided and uncertain. Of course, people like me still have to keep working, but for other people it's a day off to enjoy what little wildlife and Earth us Earthlings have left, ironically. Do you have any other questions?`

Reviewer: "Parliament building" is the preferred form.
```suggestion
			`	The attendant chuckles to himself. "I should've expected that question from your accent. Today's Earth Day, when all the cities of Earth - and I mean all of them - participate in a total blackout. Even the Parliament building was out today. It makes us think about our origins; that we once lived on the soil without electricity, all confined to just this one planet here, divided and uncertain. Of course, people like me still have to keep working, but for other people it's a day off to enjoy what little wildlife and Earth us Earthlings have left, ironically. Do you have any other questions?`
```

			choice
				`	"When did this tradition start?"`
				`	"Thank you for telling me. (Walk away.)"`
					goto pass

> 			`	"Well, my old history class tells me that the first Earth Day was in or around the 1970s, a few years after we landed on the Moon for the first time. Back then, there were a bunch of conflicts and disputes over resources and how we were fighting all over and and digging it all up to power all those dirty, primitive cars and factories and we had back then. So as a show of unity against that people started celebrating the Earth's beauty and unity by reminding us of our past."`

Reviewer 4: ```suggestion
			`	"Well, my old history class tells me that the first Earth Day was in or around the 1970s, a few years after we landed on the Moon for the first time. Back then, there were a bunch of conflicts and disputes over resources and how we were fighting all over and and digging it all up to power all those dirty, primitive cars and factories we had back then. So as a show of unity against that people started celebrating Earth's beauty and unity by reminding us of our past."`
```

> 			`	"Well, my old history class tells me that the first Earth Day was in or around the 1970s, a few years after we landed on the Moon for the first time. Back then, there were a bunch of conflicts and disputes over resources and how we were fighting all over and digging it all up to power all those dirty, primitive cars, power plants, and factories we had back then. So as a show of unity against that people started celebrating Earth's natural beauty by bringing us back to our past."`

Reviewer: ```suggestion
			`	"Well, my old history class tells me that the first Earth Day was around the 20th century, a few years after we landed on the Moon for the first time. Back then, there were a bunch of conflicts and disputes over resources and how we were fighting all over and digging it all up to power all those dirty, primitive cars, power plants, and factories we had back then. So as a show of unity against that people started celebrating Earth's natural beauty by bringing us back to our past."`
```

			`	As he finishes, the ceiling lights flash back on. "Ah, well, there you go. Too bad I missed the celebrations, but that's the end of Earth Day and my night shift. I hope you had a good Earth Day, and have a good night!"`
				decline

			label tourist
			`	You approach a woman with a bunch of oversized bags clustered around her seat. She's tapping rapidly on her datapad, but nonetheless she looks up when you approach. "Excuse me, Captain, I was just trying to figure out when the metro starts up again. Do you know?"`
			choice
				`	"I know just as much as you, sorry."`
				`	"Maybe it's to do with the blackout?"`
					goto "way in"
			`	"Ha, they shut everything down for Earth Day and then assume all us offworlders get the idea. What do you know..."`
			choice
				`	"Earth Day?"`
				`	"Shut everything down?"`

> 			`	"Didn't you see what was happening outside on your way in? It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then, it was awfully primitive times.`

Reviewer: ```suggestion
			`	"Didn't you see what was happening outside on your way in? It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then. Awfully primitive times.`
```

				goto decided

			label "way in"

> 			`	"Yes, I suppose you must have seen what's happening outside on your way in. It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then, it was awfully primitive times.`

Reviewer: ```suggestion
			`	"Yes, I suppose you must have seen what's happening outside on your way in. It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then. Awfully primitive times.`
```

			label decided

> 			`	"So they decided that, on the 22nd of April each year, all of Earth would go into total blackout to symbolize that they didn't have enough energy to sustain themselves, and to remind them that the only way to advance from this unspectacular position was to keep pushing themselves forward, find new pathways to power, lest those blackouts became longer and more frequent. Fascinating, huh?"`

Reviewer: ```suggestion
			`	"So they decided that, on the 22nd of April each year, all of Earth would go into total blackout to symbolize that they didn't have enough energy to sustain themselves, and to remind them that the only way to advance was to keep pushing themselves forward, find new pathways to power, lest those blackouts became longer and more frequent. Fascinating, huh?"`
```

			choice
				`	"It seems a bit backwards."`

> 				`	"It is fascinating, thanks. (Walk away.)"`

Reviewer: ```suggestion
				`	"It is fascinating, thanks."`
```

					goto pass

> 			`	"Well, perhaps. But Earth is like that way, sometimes. It's survived through so much, and without Earth Day to remind our ancestors that we must keep looking forward, maybe you wouldn't be in your fancy little spaceship, would you?" The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock, I've got to go now and see if the metro's opened so I can get to my accommodation. Safe travels!"`

Reviewer 4: ```suggestion
			`	"Well, perhaps. But Earth is like that sometimes. It's survived through so much, and without Earth Day to remind our ancestors that we must keep looking forward, maybe you wouldn't be in your fancy little spaceship, would you?" The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock, I've got to go now and see if the metro's opened so I can get to my accommodation. Safe travels!"`
```

> 			`	"Well, perhaps. But Earth is like that sometimes. It's survived through so much, and without Earth Day to remind our ancestors that we must keep looking forward, maybe you wouldn't be in your fancy little spaceship, would you?" The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock, I've got to go now and see if the metro's opened so I can get to my accommodation. Safe travels!"`

Reviewer: ```suggestion
			`	"Well, perhaps. But Earth is like that sometimes. It's survived through so much, and without Earth Day to remind our ancestors that we must keep looking forward, maybe you wouldn't be in your fancy little spaceship, would you?"`
			`	The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock. I've got to go now and see if the metro's opened so I can get to my accommodation. Safe travels!"`
```

				decline

			label pass

> 			`	After the clock passes midnight, the lights in the spaceport come back on, followed by the signage and lights of the city surrounding it. A few workers come and snuff out the candles before putting them away in unseen closets in maintenance corridors.`

Reviewer 3: ```suggestion
			`	After the clock passes midnight, the lights in the spaceport come back on, followed by the signage and lights of the city surrounding it. A few workers come and snuff out the candles before putting them away in unseen closets.`
```
I think that extra bit kind of kills the final impact of bustling away this grand gesture and resuming normal course.

				goto decided

			label "way in"
			`	"Yes, I suppose you must have seen what's happening outside on your way in. It's something to do with Earth Day. Back in those old days before humans were able to get off this planet, they were stuck in a chain of issues and disputes over energy. You know, they didn't have fusion reactors back then. Awfully primitive times.`

			label decided
			`	"So they decided that, on the 22nd of April each year, all of Earth would go into total blackout to symbolize that they didn't have enough energy to sustain themselves, and to remind them that the only way to advance was to keep pushing themselves forward, find new pathways to power, lest those blackouts became longer and more frequent. Fascinating, huh?"`
			choice
				`	"It seems a bit backwards."`
				`	"It is fascinating, thanks."`
					goto pass
			`	"Well, perhaps. But Earth is like that sometimes. It's survived through so much, and without Earth Day to remind our ancestors that we must keep looking forward, maybe you wouldn't be in your fancy little spaceship, would you?"`

> 			`	The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock. I've got to go now and see if the metro's opened so I can get to my accommodation. Safe travels!"`

Reviewer: ```suggestion
			`	The lights in the spaceport turn back on as she says that, and she turns to look at her data pad again. "Oh, it's twelve o'clock. I've got to see if the metro's opened so I can get to my accommodation. Safe travels!"`
```
